[2022-06-07 15:57:00] [INFO] [Recovery] 2022/06/07 - 15:57:00 panic recovered:
Get "https://localhost/version?timeout=32s": dial tcp [::1]:443: connectex: No connection could be made because the target machine actively refused it.
C:/Users/mensyli1/work/workspace/mwpath/pkg/mod/github.com/toolkits/pkg@v1.2.9/errorx/errorx.go:40 (0x16222da)
	Dangerous: }
C:/Users/mensyli1/work/workspace/mwpath/pkg/mod/github.com/toolkits/pkg@v1.2.9/ginx/errorx.go:10 (0x165616c)
	Dangerous: }
C:/Users/mensyli1/work/workspace/middleware/src/controller/postgres/postgres.go:62 (0x165615b)
	GetPostgresByName: }
C:/Users/mensyli1/work/workspace/mwpath/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165 (0x1658387)
	(*Context).Next: c.index++
C:/Users/mensyli1/work/workspace/middleware/src/util/aop/jwt.go:19 (0x165836e)
	Auth.func1: 
C:/Users/mensyli1/work/workspace/mwpath/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165 (0x16584ab)
	(*Context).Next: c.index++
C:/Users/mensyli1/work/workspace/middleware/src/util/aop/logrus.go:62 (0x1658488)
	Logrus.func1: //ÁªìÊùüÊó∂Èó¥
C:/Users/mensyli1/work/workspace/mwpath/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165 (0x165741a)
	(*Context).Next: c.index++
C:/Users/mensyli1/work/workspace/middleware/src/util/aop/recovery.go:94 (0x1657406)
	CustomRecoveryWithWriter.func1: }
C:/Users/mensyli1/work/workspace/mwpath/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165 (0x15fbb7d)
	(*Context).Next: c.index++
C:/Users/mensyli1/work/workspace/mwpath/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489 (0x15fb805)
	(*Engine).handleHTTPRequest: c.writermem.WriteHeaderNow()
C:/Users/mensyli1/work/workspace/mwpath/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445 (0x15fb364)
	(*Engine).ServeHTTP: 
C:/install/Dev/Env/Go/src/net/http/server.go:2879 (0x96459a)
	serverHandler.ServeHTTP: }
C:/install/Dev/Env/Go/src/net/http/server.go:1930 (0x95fc47)
	(*conn).serve: w.cancelCtx()
C:/install/Dev/Env/Go/src/runtime/asm_amd64.s:1581 (0x6b4b00)
	goexit: CALL	runtime¬∑goexit1(SB)	// does not return
[0m (github.com/mensylisir/kmpp-middleware/src/util/aop.CustomRecoveryWithWriter.func1.1: 82) {map[]} 
[2022-06-07 15:57:01] [ERROR] gather postgres info error: Get "https://localhost/version?timeout=32s": dial tcp [::1]:443: connectex: No connection could be made because the target machine actively refused it. (github.com/mensylisir/kmpp-middleware/src/cron/job.(*WatchPostgresqlInfo).Run.func1: 35) {map[]} 
[2022-06-07 15:57:06] [ERROR] gather postgres info error: Get "https://localhost/version?timeout=32s": dial tcp [::1]:443: connectex: No connection could be made because the target machine actively refused it. (github.com/mensylisir/kmpp-middleware/src/cron/job.(*WatchPostgresqlInfo).Run.func1: 35) {map[]} 
[2022-06-07 15:57:11] [ERROR] gather postgres info error: Get "https://localhost/version?timeout=32s": dial tcp [::1]:443: connectex: No connection could be made because the target machine actively refused it. (github.com/mensylisir/kmpp-middleware/src/cron/job.(*WatchPostgresqlInfo).Run.func1: 35) {map[]} 
[2022-06-07 15:57:16] [ERROR] gather postgres info error: Get "https://localhost/version?timeout=32s": dial tcp [::1]:443: connectex: No connection could be made because the target machine actively refused it. (github.com/mensylisir/kmpp-middleware/src/cron/job.(*WatchPostgresqlInfo).Run.func1: 35) {map[]} 
[2022-06-07 15:57:21] [ERROR] gather postgres info error: Get "https://localhost/version?timeout=32s": dial tcp [::1]:443: connectex: No connection could be made because the target machine actively refused it. (github.com/mensylisir/kmpp-middleware/src/cron/job.(*WatchPostgresqlInfo).Run.func1: 35) {map[]} 
[2022-06-07 15:57:26] [ERROR] gather postgres info error: Get "https://localhost/version?timeout=32s": dial tcp [::1]:443: connectex: No connection could be made because the target machine actively refused it. (github.com/mensylisir/kmpp-middleware/src/cron/job.(*WatchPostgresqlInfo).Run.func1: 35) {map[]} 
[2022-06-07 15:57:31] [ERROR] gather postgres info error: Get "https://localhost/version?timeout=32s": dial tcp [::1]:443: connectex: No connection could be made because the target machine actively refused it. (github.com/mensylisir/kmpp-middleware/src/cron/job.(*WatchPostgresqlInfo).Run.func1: 35) {map[]} 
[2022-06-07 15:57:36] [ERROR] gather postgres info error: Get "https://localhost/version?timeout=32s": dial tcp [::1]:443: connectex: No connection could be made because the target machine actively refused it. (github.com/mensylisir/kmpp-middleware/src/cron/job.(*WatchPostgresqlInfo).Run.func1: 35) {map[]} 
[2022-06-07 15:57:41] [ERROR] gather postgres info error: Get "https://localhost/version?timeout=32s": dial tcp [::1]:443: connectex: No connection could be made because the target machine actively refused it. (github.com/mensylisir/kmpp-middleware/src/cron/job.(*WatchPostgresqlInfo).Run.func1: 35) {map[]} 
[2022-06-07 15:57:46] [ERROR] gather postgres info error: Get "https://localhost/version?timeout=32s": dial tcp [::1]:443: connectex: No connection could be made because the target machine actively refused it. (github.com/mensylisir/kmpp-middleware/src/cron/job.(*WatchPostgresqlInfo).Run.func1: 35) {map[]} 
[2022-06-07 15:57:51] [ERROR] gather postgres info error: Get "https://localhost/version?timeout=32s": dial tcp [::1]:443: connectex: No connection could be made because the target machine actively refused it. (github.com/mensylisir/kmpp-middleware/src/cron/job.(*WatchPostgresqlInfo).Run.func1: 35) {map[]} 
[2022-06-07 15:57:56] [ERROR] gather postgres info error: Get "https://localhost/version?timeout=32s": dial tcp [::1]:443: connectex: No connection could be made because the target machine actively refused it. (github.com/mensylisir/kmpp-middleware/src/cron/job.(*WatchPostgresqlInfo).Run.func1: 35) {map[]} 
